pkg/standalone/list.go

```diff
@@ -166,24 +118,17 @@ func List() ([]ListOutput, error) {
 			listRow := ListOutput{
 				Created: createTime.Format("2006-01-02 15:04.05"),
 				Age:     age.GetAge(createTime),
-				PID:     proc.Pid(),
+				PID:     cliPID,
```

Comment:
The PID here is only `proc.Pid()` if there is an error parsing the line 119 https://github.com/dapr/cli/pull/763/files#diff-40b8b5acdd5260df8de205a924b446eb2483901d4e4c30c6a23932d3223e99e9R117

But do we not want this to always be the proc.Pid()? 
@artursouza @wcs1only @yaron2 thoughts?

Reply:
I think the logic is that the PID should be the PID of the `dapr` application and not the sidecar if a parent `dapr` process exists. If there is no parent `dapr` process running (i.e. you've run an independent sidecar), the cliPID throws an error since there is no parent process. In this case, the PID is set to proc.Pid() since only the sidecar is running

Reply:
Hmm, we probably want to disambiguate what PID we're talking about. Giving the CLI pid when the CLI is being used and the daprd PID when it is not has the potential to confuse our users quite a bit. Maybe the solution is to have two columns in the table, one for CLI PID and one for daprd PID. We can leave the CLI pid column blank. Or the other option would be to always give the daprd pid.

Reply:
I talked to @artursouza when first implementing this, and he had said that using the sidecar PID if there was no parent CLI process should be okay. But if that's too ambiguous, I think specifying both PIDs could help with that

Reply:
Put yourself in the shoes of the user, when they see:
```
  APP ID     HTTP PORT  GRPC PORT  APP PORT  COMMAND         AGE  CREATED              PID    
  pythonapp  61845      61846      0         python3 app.py  7s   2021-08-04 11:51.00  37011  
  nodeapp    3500       61800      3000      node app.js     13s  2021-08-04 11:50.54  36990  
```

What are they to think that PID means? It could actually mean three different things, because there are three processes running (is it the CLI, daprd, or my app's PID?). This is somewhat ambiguous to begin with, but it is even more ambiguous if the answer to that question changes depending on how daprd was launched.  

I'm personally leaning towards just changing that column name to "DAPRD PID" and always giving the daprd pid.

Reply:
I originally said to give both the CLI and daprd PID because other dapr commands (mainly `dapr stop`) use the CLI PID. Compatibility with the stop command was also part of the reasoning behind setting the PID to be the daprd sidecar's PID if there was an independent one running. I think the Windows stop code specifically needs the CLI PID not the daprd PID so just using the daprd PID might not be the best approach

But if killing the daprd sidecar also kills the CLI, then maybe just showing the daprd PID makes most sense.

Reply:
`dapr stop` should also stop the app if it was started as part of `dapr run`. I am OK to remove ambiguity here. Maybe we do need to have 3 columns: CLI PID, Sidecar PID and App PID and they are not always present. Then, `dapr stop` must change its logic to know how to stop on each combination of these 3 PIDs being present or not.

Reply:
I added a column for the sidecar PID and updated the stop commands to use the sidecar PID when there is no parent CLI process running. Not sure if we also want to have the App PID in there as well, but I feel like that might be unnecessary information. @wcs1only @artursouza